spring-5-security/src/main/resources/persistence.properties

```diff
@@ -0,0 +1,11 @@
+# jdbc.X
+jdbc.driverClassName=org.h2.Driver
+jdbc.url=jdbc:h2:mem:db;DB_CLOSE_DELAY=-1;INIT=CREATE SCHEMA IF NOT EXISTS USERS
+jdbc.user=sa
+jdbc.pass=sa
```

Comment:
this file should not be needed, spring boot should be able to cover all of this automatically, even if any property is still needed it should be added to application.properties with a spring boot equivalent of these properties for e.g. spring.datasource.username instead of jdbc.user, for more reference - https://docs.spring.io/autorepo/docs/spring-boot/1.3.0.M2/reference/html/boot-features-sql.html